docs/export_python_type.md

```diff
@@ -0,0 +1,96 @@
+## Exporting Nim types as Python classes
+
+Warning! This is experimental.
+* An exported type should be a ref object and inherits `PyNimObjectExperimental` directly or indirectly.
+* The type will only be exported if at least one exported "method" is defined.
+* A proc will be exported as python type method *only* if it's first argument is of the corresponding type and is called `self`. If the first argument is not called `self`, the proc will exported as a global module function.
+
+#### Simple Example
+```nim
+# mymodule.nim
+type TestType = ref object of PyNimObjectExperimental
+  myField: string
+
+proc setMyField(self: TestType, value: string) {.exportpy.} =
+  self.myField = value
+
+proc getMyField(self: TestType): string {.exportpy.} =
+  self.myField
+```
+
+``` py
+# test.py
+import mymodule
+tt = mymodule.TestType()
+tt.setMyField("Hello")
+assert(tt.getMyField() == "Hello")
+```
+
+#### `__init__`, `__del__`, and `__repr__`
+* [example](../tests/export_pytype.nim)
+```nim
+# simple.nim
+## compile as simple.so
+
+import nimpy
+import strformat
+
+pyExportModule("simple") # only needed if your filename is not simple.nim
+
+type
+    SimpleObj* = ref object of PyNimObjectExperimental
+        a* : int
+
+## if 
+##    1) the function name is like `init##TypeName`
+##    2) there is only one argument
```

Comment:
Isn't init allowed to have more than one argument?

Reply:
right; changed to --> there is at least one argument